hack/test/e2e-iso.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eou pipefail
+
+qemu-img create -f qcow2 disk.qcow2 4G
+
+qemu-system-x86_64 \
+  -m 2048 \
+  -nographic \
+  -serial mon:stdio \
+  -enable-kvm \
+  -cpu host \
+  -boot cd \
+  -netdev user,id=talos,ipv4=on,net=10.254.0.0/24,dhcpstart=10.254.0.10,hostfwd=tcp::4443-:443,hostfwd=tcp::50000-:50000,hostfwd=tcp::6443-:6443,hostname=control-plane-1 \
```

Comment:
A little worried about this one since it has the potential to clash with another run.

Reply:
we're inside a k8s pod sandbox anyways?